apps/pie-docs/src/components/checkbox/motion/motion.md

```diff
@@ -0,0 +1,86 @@
+---
+eleventyNavigation:
+    key: Motion
+    parent: 'Checkbox'
+    order: 2
+shouldShowContents: true
+eleventyComputed:
+    hover: "{% include './hover.json' %}"
+    active: "{% include './active.json' %}"
+    selected: "{% include './selected.json' %}"
+    unselected: "{% include './unselected.json' %}"
+---
+
+## Example
+
+Illustrates the animations that happen in the checkbox component. In this component we have animations on hover and active states as well as when the component is selected and unselected.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/checkbox/motion-example.svg",
+    alt: "Three checkboxes in a horizontal row. First one in default state, second one in hover state and third one in checked state.",
+    width: "586px",
+    height: "m"
+} %}
+
+---
+
+## Hover
+
+When the user hovers over an unselected checkbox (on:Hover), the hover state $hover01 fades in at 200ms. When a user moves the mouse away (off:Hover) the hover state $hover01 fades out at 200ms.
```

Comment:
is `(on:Hover)` our way of indicating browser events? I am not sure if we've agreed on a convention for this before but it's certainly an interesting idea if so. I am unsure what exactly it adds here personally but keen to know what others think.

Reply:
I'm not sure who it's aimed for but to my eye it looks like `:hover` but with incorrect casing 😅

Reply:
yes, agreed 😂